File: csharp/DVSClient/DVSClient/server/address/layout/RestApiAddressLayoutLineElement.cs
using DVSClient.Address.Layout.Elements;
using Newtonsoft.Json;

namespace DVSClient.Server.Address.Layout
{
    public class RestApiAddressLayoutLineElement
    {
        [JsonProperty("element_name")]
        public string? ElementName { get; set; }

        public RestApiAddressLayoutLineElement() { }

        public RestApiAddressLayoutLineElement(IAddressElement? addressElement)
        {
            ElementName = addressElement?.GetElementName();
        }
    }
}

File: java/src/main/java/com/experian/dvs/client/server/address/layout/RestApiAddressLayoutLineElement.java
package com.experian.dvs.client.server.address.layout;

import com.experian.dvs.client.address.layout.elements.AddressElement;
import com.fasterxml.jackson.annotation.JsonProperty;

public class RestApiAddressLayoutLineElement {

    @JsonProperty("element_name")
    private String elementName;

    public RestApiAddressLayoutLineElement() {
    }

    public RestApiAddressLayoutLineElement(final AddressElement addressElement) {
        this.elementName = addressElement.getElementName();
    }

    public String getElementName() {
        return elementName;
    }

    public void setElementName(String elementName) {
        this.elementName = elementName;
    }
}
